lib/CurrencyExchangeRateFields/CurrencyExchangeRateFields.js

```diff
@@ -146,7 +146,7 @@ const CurrencyExchangeRateFields = ({
               readOnly={!isExchangeRateEnabled}
               tooltipText={tooltipTextExchangeRate}
               required={isExchangeRateRequired}
-              validate={isExchangeRateRequired ? validateRequired : undefined}
+              validate={isExchangeRateRequired ? validateRequiredPositiveAmount : undefined}
```

Comment:
is it expected to skip "positive amount" validation if the field is not required?

Reply:
Correct. If the user wants to set a custom exchange rate then the required checkbox should be checked

Reply:
sounds strange

Reply:
when field is not required you just need to validate when value is populated, introduce another validation method (if doesn't exist) validatePositiveAmount

Reply:
https://github.com/user-attachments/assets/b7e42a18-6317-4283-a60d-36348ba769f0